sdk/automanage/Azure.ResourceManager.Automanage/src/Generated/BestPracticeVersionResource.cs

```diff
@@ -0,0 +1,137 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Globalization;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Core;
+using Azure.Core.Pipeline;
+using Azure.ResourceManager;
+
+namespace Azure.ResourceManager.Automanage
+{
+    /// <summary>
+    /// A Class representing a BestPracticeVersion along with the instance operations that can be performed on it.
+    /// If you have a <see cref="ResourceIdentifier" /> you can construct a <see cref="BestPracticeVersionResource" />
+    /// from an instance of <see cref="ArmClient" /> using the GetBestPracticeVersionResource method.
+    /// Otherwise you can get one from its parent resource <see cref="BestPracticeResource" /> using the GetBestPracticeVersion method.
+    /// </summary>
+    public partial class BestPracticeVersionResource : ArmResource
```

Comment:
Each resource should have CRUD tests associated with it

Reply:
Used directives to exclude some of the operations that are not quite customer ready.